docs/csharp/language-reference/operators/arithmetic-operators.md

```diff
@@ -79,7 +79,7 @@ The unary `+` operator returns the value of its operand. The unary `-` operator
 
 [!code-csharp-interactive[unary plus and minus](~/samples/csharp/language-reference/operators/ArithmeticOperators.cs#UnaryPlusAndMinus)]
 
-The unary `-` operator doesn't support the [ulong](../builtin-types/integral-numeric-types.md) type.
+The [ulong](../builtin-types/integral-numeric-types.md) type doesn't support the unary `-` operator.
 
 ## Multiplication operator *
```

Comment:
I'm not sure if this should be escaped ?

```suggestion
## Multiplication operator \*
```

Reply:
No escape necessary, it renders well as it is now